Packs/CyberArkEPV/README.md

```diff
@@ -0,0 +1,43 @@
+
+# CyberArk Enterprise Password Vault
+This pack includes Cortex XSIAM content.
+
+
+## Configuration on Server Side
+Syslog messages can be sent to multiple syslog servers in two different ways.
+- One message can be sent to multiple servers by configuring an XSLT file.
+- Multiple messages can be sent to different servers and formatted differently for each server by configuring multiple XSLT files, formats, and code-message lists. The code-message lists must be matched. They must contain the same number of items in the same order.
+```http
+  Note: The .ini file contains these configuration values.
```

Comment:
Not sure what configuration values you are referring to here. Do you mean the values in Step 1?

Reply:
Hi Shirley,
Yes